statedb_test.go

```diff
@@ -180,8 +271,8 @@ func insertHeaderCID(db *pgxpool.Pool) error {
 	coinbase
 ) VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14)`
 	_, err := db.Exec(testCtx, sql,
-		BlockNumber.Uint64(),
-		BlockHash.String(),
+		blockNumber,
+		blockHash,
 		BlockParentHash.String(),
 		HeaderCID.String(),
```

Comment:
Let's also hardcode these to zero values so it's clear they're just filler values for now
```suggestion
		common.HexToHash("0").String(), "",
```

Reply:
Word! Thinking we should make variables for the incremented block numbers too

Reply:
Gonna use `Header.UncleHash.String()`